base/bucket_gocb.go

```diff
@@ -898,23 +843,22 @@ func (bucket CouchbaseBucketGoCB) Update(k string, exp int, callback sgbucket.Up
 	maxCasRetries := 100000 // prevent infinite loop
 	for i := 0; i < maxCasRetries; i++ {
 
-		var value interface{}
+		var value []byte
 		var err error
 
 		// Load the existing value.
 		// NOTE: ignore error and assume it's a "key not found" error.  If it's a more
 		// serious error, it will probably recur when calling other ops below
 
 		gocbExpvars.Add("Update_Get", 1)
-		cas, _ := bucket.Get(k, &value)
-
-		var callbackParam []byte
-		if value != nil {
-			callbackParam = value.([]byte)
+		cas, err := bucket.Get(k, &value)
```

Comment:
Do we want to call bucket.GetRaw here instead?   Particularly with the (temporary) fix that's mentioned in GetRaw?